src/components/Amount.vue

```diff
@@ -1,25 +1,36 @@
 <template>
     <span class="amount" :class="{ approx: showApprox && isApprox }">
         {{ formattedAmount }}
-        <span class="nim">NIM</span>
+        <span class="currency" :class="currency">{{currency}}</span>
     </span>
 </template>
 
 <script lang="ts">
-import { Component, Prop, Vue } from 'vue-property-decorator';
+import { Component, Prop, Vue, Watch } from 'vue-property-decorator';
+import { formatNumber, moveComma } from '@nimiq/utils';
+import bigInt from 'big-integer';
```

Comment:
You can import this as `type BigInteger = import('big-integer').BigInteger;` to ensure that only the type is imported without bundling the lib.